lib/wash/entry.rb

```diff
@@ -343,6 +354,50 @@ def cache_ttls(ttls = {})
       @cache_ttls = @cache_ttls.merge(ttls)
     end
 
+    # volumefs returns an entry that provides a view of the parent entry's filesystem.
+    # Use it when listing children. It can be used with `prefetch :list`.
```

Comment:
I'd probably just say `volumefs` creates the `volume::fs` entry as specified in the external plugin docs. The current wording here makes it seem like a `wash-ruby` thing when it is more general than that.

Also, is mentioning `prefetch :list` necessary? Maybe a nit on my end, but when I read that, my brain forgets that this is just a regular entry JSON. I think your example is clear enough and shows that you can use `prefetch :list` with it. It might be nice to add one more entry in the `list` call just so "volumefs is a regular entry" is really codified